src/pages/Home/Banner/index.tsx

```diff
@@ -15,9 +16,12 @@ const _Banner: FC<{ latestBlock?: State.Block }> = ({ latestBlock }) => {
     const container = ref.current
     if (!container) return
     try {
-      const r = createBannerRender(container)
-      setRender(r)
-      return () => r.destroy()
+      const testRenderTime = renderTinyTestScene(container)
```

Comment:
We need to provide a brief explanation here to express the necessity of this code snippet and leave a link to the related issue: https://github.com/Magickbase/ckb-explorer-public-issues/issues/218.